tsconfig.json

```diff
@@ -16,5 +16,6 @@
     "jsx": "preserve",
     "baseUrl": "src"
   },
-  "include": ["src"]
+  "include": ["src"],
+  "exclude": ["src/utils/tests/resources"]
```

Comment:
These resource files should be removed since the tests shouldn't be using the file system